packages/core/src/App/Components/Layout/Header/menu-links.jsx

```diff
@@ -3,45 +3,79 @@ import React from 'react';
 import { Text } from '@deriv/components';
 import { BinaryLink } from '../../Routes';
 
-const MenuLinks = ({ is_logged_in, items }) => (
+const MenuLinks = ({ is_logged_in, items, is_pre_appstore }) => (
     <React.Fragment>
         {!!items.length && (
             <div className='header__menu-links'>
                 {items.map((item, idx) => {
                     const item_text = item.text();
 
-                    return item.login_only && item.login_only !== is_logged_in ? null : (
-                        <BinaryLink
-                            id={item.id}
-                            key={idx}
-                            to={item.link_to || undefined}
-                            onClick={item.onClick || undefined}
-                            href={item.href || undefined}
-                            className='header__menu-link'
-                            active_class='header__menu-link--active'
-                        >
-                            <React.Fragment>
-                                {item_text && (
-                                    <Text
-                                        size='m'
-                                        line_height='xs'
-                                        title={item_text}
-                                        className='header__menu-link-text'
-                                    >
-                                        {item.icon}
-                                        {item_text}
-                                        {item.logo}
-                                    </Text>
-                                )}
-                                {item.image && (
-                                    <span className='header__menu-link-text'>
-                                        {item.image}
-                                        {item.logo}
-                                    </span>
-                                )}
-                            </React.Fragment>
-                        </BinaryLink>
-                    );
+                    return item.login_only && item.login_only !== is_logged_in
+                        ? null
+                        : (!is_pre_appstore && (
+                              <BinaryLink
+                                  id={item.id}
+                                  key={idx}
+                                  to={item.link_to || undefined}
+                                  onClick={item.onClick || undefined}
+                                  href={item.href || undefined}
```

Comment:
Any reason to pass `undefined` ? If a variable is not declared with value it will be `undefined` by default

Reply:
Nope. I just followed the previous code. Will refactor it